policyengine-client/src/countries/us/us.jsx

```diff
@@ -690,9 +690,9 @@ export class US extends Country {
 
     setHouseholdMaritalStatus(status) {
         let situation = this.situation;
-        if (status === "Married" & !situation.people.includes("Your spouse")) {
+        if (status === "Married" & !situation.people.hasOwnProperty("Your spouse")) {
```

Comment:
Thanks for making a start on this @mwufi! I just realised the true cause of the bug: bracket application on these two lines (the first condition should be bracketed on its own). Hope you don't mind my editing the PR and merging.

```suggestion
        if ((status === "Married") & !situation.people.includes("Your spouse")) {
```